client/web/compose/src/components/ModuleFields/Editor/Bool.vue

```diff
@@ -41,7 +53,7 @@
     </c-input-checkbox>
 
     <div
-      v-if="!field.options.switch"
+      v-if="!field.options.switch && !options.horizontalFieldLayoutEnabled"
```

Comment:
You dont need the extra check since the only thing that matters is the first condition